### ISSUE TITLE
BUG Router Parameter in windows only 

### ISSUE THREAD

**Issue by malikkubs** [kodepandai/lunox-framework#16]
kalo  memakai router seperti loacalhost:8000/:slug di linux, css styling tidak dapat berjalan, dan di windows menampilkan halaman 404

sedangkan memakai router seperti localhost:8000/test/slug di linux dapat menampiklan halaman secara sempurna, sedangkan di windows menampilkan 404 bukan 303 ye.

**Comment by axmad386** [comment]
confirmed. Can be reproduced on windows only
example:
```js
Route.get('/article/:slug', ()=>'OK');
```
will return `OK` in linux but `404` on windows.
Thanks for report @malikkubs

**Comment by axmad386** [comment]
Fixed in [v1.7.1](https://github.com/kodepandai/lunox-framework/releases/tag/v1.7.1)
please update lunox framework to the latest version.
```
pnpm add lunox@latest
```
